### PR TITLE
[ja] Linear algebra

### PR DESCRIPTION
**Pull request by RobAltena**
WIP. first commit.

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you for all your work @RobAltena! Feel free to let us know when you are ready for the review process.

**Comment by RobAltena** [comment]
@shervinea Ready for review.

**Comment by shervinea** [comment]
Thank you for your work @RobAltena! Let's start the review process.

File: ja/refresher-linear-algebra.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+&#10230;
+ノルムは関数N:V⟶[0,+∞[　Vはベクトル空間、　すべてのx、y∈Vについて：
+]]
+<br>
+
+**35. N(ax)=|a|N(x) for a scalar**
+
+&#10230;
+N(ax)=|a|N(x) スカラー用
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]: Looking at the pdf, it appears that "a" in "for a scalar" refers to the variable a.
```suggestion
スカラー a に対して N(ax)=|a|N(x) 
```

File: ja/refresher-linear-algebra.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**36. if N(x)=0, then x=0**
+
+&#10230;
+N（x）= 0の場合、x = 0
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
N（x）= 0ならば x = 0
```

File: ja/refresher-linear-algebra.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**37. For x∈V, the most commonly used norms are summed up in the table below:**
+
+&#10230;
+x∈V、一般的に使用されるノルムは、以下の表にまとめられています。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
x∈Vに対して、最も多用されているノルムは、以下の表にまとめられています。
```

File: ja/refresher-linear-algebra.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+**38. [Norm, Notation, Definition, Use case]**
+
+&#10230;
+[ノルム、表記法、定義、使用事例]
+<br>
+
+**39. Linearly dependence ― A set of vectors is said to be linearly dependent if one of the vectors in the set can be defined as a linear combination of the others.**
+
+&#10230;
+線形依存 ― ベクトルの集合は、その集合内のベクトルのうちの1つが他のベクトルの線形結合として定義できる場合、線形従属であると言われます。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
線形従属 ― ベクトルの集合に対して、少なくともどれか一つのベクトルを他のベクトルの線形結合として定義できる場合、その集合が線形従属であるといいます。
```
or perhaps "他のベクトルの線形結合で表すことができる" or something?

File: ja/refresher-linear-algebra.md [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**40. Remark: if no vector can be written this way, then the vectors are said to be linearly independent**
+
+&#10230;
+備考：この方法でベクトルを書くことができない場合、ベクトルは線形独立していると言われます。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]: ベクトル集合にこのようなベクトルの表し方ができるベクトルが存在しない場合は、線形独立だといいます。

File: ja/refresher-linear-algebra.md [hunk 6]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**41. Matrix rank ― The rank of a given matrix A is noted rank(A) and is the dimension of the vector space generated by its columns. This is equivalent to the maximum number of linearly independent columns of A.**
+
+&#10230;
+行列の階数　―　行列Aの階数をrank（A）と表記します。　それはその列によって生成されたベクトル空間の次元です。これは、Aの線形独立列の最大数に相当します。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
行列の階数　―　行列Aの階数は rank（A）と表記し、列空間の次元を表します。これは、Aの線形独立の列の最大数に相当します。
```

File: ja/refresher-linear-algebra.md [hunk 7]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**42. Positive semi-definite matrix ― A matrix A∈Rn×n is positive semi-definite (PSD) and is noted A⪰0 if we have:**
+
+&#10230;
+半正定値行列 ― 以下の式が成り立つとき、行列 A∈Rn×n、　A⪰0 は半正定値(PSD)
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
半正定値行列 ― 行列 A, A∈Rn×nに対して、以下の式が成り立つならば、 Aを半正定値(PSD)といい、A⪰0と表記します。
```

File: ja/refresher-linear-algebra.md [hunk 8]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**43. Remark: similarly, a matrix A is said to be positive definite, and is noted A≻0, if it is a PSD matrix which satisfies for all non-zero vector x, xTAx>0.**
+
+&#10230;
+備考：　同様に、行列Ａは、正定値行列であると言われ、A≻0、それが全ての非ゼロベクトルを満足するＰＳＤ行列である場合と表記される。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
備考：　同様に、全ての非ゼロベクトルx, xTAx>0に対して条件を満たすような行列Aは正定値行列といい、A≻0と表記します。
```

File: ja/refresher-linear-algebra.md [hunk 9]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**44. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
+
+&#10230;
+固有値、固有ベクトル　―　与えられた行列A∈Rn×n。以下の式が成り立つとき、もしベクトルz∈Rn∖{0}、固有ベクトルと呼ばれる、が存在する場合ならばλはAの固有値であると言われる：
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]: This was a hard one! I'm not too confident with my translation...
```suggestion
固有値、固有ベクトル　―　行列 A, A∈Rn×nに対して、以下の条件を満たすようなベクトルz, z∈Rn∖{0}が存在するならば、λは固有値といい、z は固有ベクトルといいます。
```

File: ja/refresher-linear-algebra.md [hunk 10]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**45. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
+
+&#10230;
+スペクトル定理 ― A∈Rn×nとする。　Aが対称ならば、Aは実直交行列U∈Rn×nによって対角化可能です。Λ=diag(λ1,...,λn)と書くと、次のようになります。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
スペクトル定理 ― A∈Rn×nとします。　Aが対称ならば、Aは実直交行列U∈Rn×nによって対角化可能です。Λ=diag(λ1,...,λn)と表記すると、次のように表現できます。

File: ja/refresher-linear-algebra.md [hunk 11]
```diff
@@ -0,0 +1,342 @@
...
+**46. diagonal**
+
+&#10230;
+対角
+<br>
+
+**47. Singular-value decomposition ― For a given matrix A of dimensions m×n, the singular-value decomposition (SVD) is a factorization technique that guarantees the existence of U m×m unitary, Σ m×n diagonal and V n×n unitary matrices, such that:**
+
+&#10230;
+特異値分解 ― Aをm×nの行列とする。特異値分解（SVD）は、Ｕ ｍ×ｍのユニタリ行列、ｍ ｍ×ｎの対角行列、およびＶ ｎ×ｎのユニタリ行列の存在を保証する因数分解手法である、次のようになります。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
特異値分解 ― Aをm×nの行列とします。特異値分解（SVD）は、ユニタリ行列Ｕ ｍ×ｍ、Σ ｍ×ｎの対角行列、およびユニタリ行列Ｖ ｎ×ｎの存在を保証する因数分解手法で、以下の条件を満たします。
```

File: ja/refresher-linear-algebra.md [hunk 12]
```diff
@@ -0,0 +1,342 @@
...
+**48. Matrix calculus**
+
+&#10230;
+行列微積分
+<br>
+
+**49. Gradient ― Let f:Rm×n→R be a function and A∈Rm×n be a matrix. The gradient of f with respect to A is a m×n matrix, noted ∇Af(A), such that:**
+
+&#10230;
+勾配 ― f:Rm×n→Rを関数とし、A∈Rm×nを行列とする。 Aに対するfの勾配はm×n行列で、∇Af（A）と表記され。次のように：
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
勾配 ― f:Rm×n→Rを関数とし、A∈Rm×nを行列とします。 Aに対するfの勾配はm×n行列で、∇Af（A）と表記し、次の条件を満たします。
```

File: ja/refresher-linear-algebra.md [hunk 13]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**50. Remark: the gradient of f is only defined when f is a function that returns a scalar.**
+
+&#10230;
+備考：　fの勾配は、fがスカラーを返す関数である場合にのみ定義されます。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
備考：　fの勾配は、fがスカラーを返す関数であるときに限り存在します。
```

File: ja/refresher-linear-algebra.md [hunk 14]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**51. Hessian ― Let f:Rn→R be a function and x∈Rn be a vector. The hessian of f with respect to x is a n×n symmetric matrix, noted ∇2xf(x), such that:**
+
+&#10230;
+ヘッセ行列 ― f：Rn→Rを関数とし、x∈Rnをベクトルとする。 xに関するfのヘッセ行列は、次のように∇2xf（x）と表記されるn×n対称行列です。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
ヘッセ行列 ― f：Rn→Rを関数とし、x∈Rnをベクトルとします。 xに対するfのヘッセ行列は、n×n対称行列で∇2xf（x）と表記し、以下の条件を満たします。
```

File: ja/refresher-linear-algebra.md [hunk 15]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**52. Remark: the hessian of f is only defined when f is a function that returns a scalar**
+
+&#10230;
+備考：　fのヘッセ行列は、fがスカラーを返す関数である場合にのみ定義されます。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
備考：　fのヘッセ行列は、fがスカラーを返す関数である場合に限り存在します。
```

File: ja/refresher-linear-algebra.md [hunk 16]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**53. Gradient operations ― For matrices A,B,C, the following gradient properties are worth having in mind:**
+
+&#10230;
+勾配演算 ― 行列A、B、Cの場合、次の勾配特性があります。
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
勾配演算 ― 行列A、B、Cの場合、特に以下の勾配の性質を意識する甲斐があります。
```

File: ja/refresher-linear-algebra.md [hunk 17]
```diff
@@ -0,0 +1,342 @@
...
+<br>
+
+**54. [General notations, Definitions, Main matrices]**
+
+&#10230;
+[表記, 定義, 主行列]
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
[表記, 定義, 主な行列の種類]
```

File: ja/refresher-linear-algebra.md [hunk 18]
```diff
@@ -0,0 +1,342 @@
...
+**56. [Matrix properties, Norm, Eigenvalue/Eigenvector, Singular-value decomposition]**
+
+&#10230;
+[行列特性, 行列ノルム, 固有値/固有ベクトル, 特異値分解]
+<br>
+
+**57. [Matrix calculus, Gradient, Hessian, Operations]**
+
+&#10230;
+[行列計算, 勾配, ヘッセ行列, 演算]
```
Comment (Kamulau) [inline ja/refresher-linear-algebra.md]:
```suggestion
[行列微積分, 勾配, ヘッセ行列, 演算]
```

**Comment by RobAltena** [comment]
@shervinea Review is complete.

**Comment by shervinea** [comment]
Thank you @RobAltena and @Kamulau!

**Pull request merged**